pages/en/committee.markdown

```diff
@@ -7,9 +7,9 @@ toc_sticky: true
 lang: en
 ---
 ## Caleb Kibet (Chair)
-![Add Profile Photo](https://sbcf.fr/wp-content/uploads/2018/03/sbcf-default-avatar.png)
+![Add Profile Photo](https://www.h3abionet.org/images/2019/nodes/45_photo_.jpg)
```

Comment:
```suggestion
![Photo of Caleb Kibet]({{ site.baseurl }}/assets/images/45_photo_.jpg){: width="150" height="150"}
```
- Comment is helpful for people using screen readers. 
- Path to image assumes it has been uploaded to https://github.com/kipkurui/BOSS22/tree/patch-1/assets/images